storage/innobase/dict/dict0stats.cc

```diff
@@ -1514,7 +1585,7 @@ dict_stats_analyze_index_level(
 		/* increment the pages counter at the end of each page */
 		if (rec_is_last_on_page) {
 
-			(*total_pages)++;
+			(stats->total_pages)++;
```

Comment:
The parentheses are redundant and should be removed. (Originally we could have written `++*total_pages` without parentheses.)